open option not working

## Type of Change
- [ ] New Feature Request
- [ ] Documentation
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
After the latest release (0.5.0) the `--open` flag is not working.
```
$ npx git-explorer --port=4200 --open
npx: installed 93 in 6.176s
Cannot find module 'open'
```

## Details
Related to #7 , the issue / solution seems pretty straightforward.  The **open** package needs to be a `dependency`.  Currently it is a `devDependency` in _package.json_.
```json
{
  ...
  
   "dependencies": {
    ...
    "yargs": "^15.0.2"
  },
  "devDependencies": {
    "copy": "^0.3.2",
    "eslint": "^6.7.1",
    "open": "^7.0.0",
    "rimraf": "^3.0.0"
  }
}
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "git-explorer",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "description": "A local development tool that provides offline-first support for previewing and comparing branches in a local git repository.",
   "author": "Owen Buckley <owen@thegreenhouse.io>",
   "license": "MIT",

--- a/package.json
+++ b/package.json
@@ -41,13 +41,13 @@
     "express-favicon": "^2.0.1",
     "lit-element": "^2.2.1",
     "lit-html": "^1.1.2",
+    "open": "^7.0.0",
     "simple-git": "^1.126.0",
     "yargs": "^15.0.2"
   },
   "devDependencies": {
     "copy": "^0.3.2",
     "eslint": "^6.7.1",
-    "open": "^7.0.0",
     "rimraf": "^3.0.0"
   }
 }

